velox/functions/lib/MapFromEntries.cpp

```diff
@@ -234,11 +247,15 @@ class MapFromEntriesFunction : public exec::VectorFunction {
     checkDuplicateKeys(mapVector, *remianingRows, context);
```

Comment:
Not your change. But could you help fix the typo: remianingRows -> remainingRows?